requirements_flask_app.txt

```diff
@@ -1,3 +1,4 @@
 flask
 flask_cors
 poetry
+argon2-cffi
```

Comment:
Before merging at all this'll need to be pip installed onto the server.

Reply:
Understood. Resolve this conversation when you are done with this.

Reply:
it was already installed? interesting. well merging time I guess.